src/libs/state.ts

```diff
@@ -136,7 +136,7 @@ lastActiveTimeStore.update((v) => v || {});
 export const toggleStateAtom = staticStorageSlot(getSessionStorage(), 'toggleState');
 toggleStateAtom.update((v) => v || { notebooksTab: true });
 
-export const azurePreviewStore = staticStorageSlot(getLocalStorage(), 'azurePreview');
+export const azurePreviewStore: Atom<any> = staticStorageSlot(getLocalStorage(), 'azurePreview');
```

Comment:
Can this be typed more specifically?

```suggestion
export const azurePreviewStore: Atom<boolean> = staticStorageSlot(getLocalStorage(), 'azurePreview');
```